docs/bok/Artifacts/osrb.md

```diff
@@ -0,0 +1,74 @@
+## Creating an Open Source Review Board
+
+Creating an Open Source Review Board (sometimes known as "Advisory Councils) for any open source contributions coming outside of a FINTECH enterprise involves establishing a structured group responsible for evaluating and guiding open source contributions made by the company's employees. This board ensures that these contributions align with the company's strategic interests, comply with legal and regulatory standards, risk and security standards as well as foster positive relationships within the open source community. 
+
+**Read below for a structured approach to forming such a board**
+
+### Define the Purpose and Scope
+
+**Objective**
+- Clarify the review board's primary goal, which could include ensuring that open source contributions enhance the company's reputation, comply with legal standards, and align with its strategic goals. 
+
+**Scope**
+- Determine the types of projects or contributions that require review. This might encompass code contributions/enhancements, documentation, participation in open source project governance, starting new open source projects or taking these open source projects into foundation incubation.
+
+### Establish Governance and Policies 
+
+**Composition** 
+- The board should include members from diverse backgrounds, such as legal experts specializing in intellectual property and open source licensing, software engineers with experience in open source projects, security experts, risk and representatives from product and strategy teams. 
```

Comment:
might be best to provide a bullet-list here that people can check off.  Does cover the bases though